app/controllers/account/verifications_controller.rb

```diff
@@ -0,0 +1,114 @@
+# frozen_string_literal: true
+
+module Account
+  class VerificationsController < ApplicationController
+    before_action :login_required, except: [
+      :new,
+      :create,
+      :reverify,
+      :resend_email
+    ]
+    before_action :disable_link_prefetching, except: [
+      :new,
+      :create
+    ]
+
+    # Regular signup verifications hit this page only (via GET).
```

Comment:
Thank you for the comments!  I would have no clue what the difference between account/login/new and account/login/create would be otherwise!